### DISCUSSION TITLE
已经在 cloudflare 部署了一个 1Panel 后续我们会持续观察是否有该问题

### DISCUSSION THREAD

**Discussion by likesrt** [1Panel-dev/1Panel#5066] [Q&A]
已经在 cloudflare 部署了一个 1Panel 后续我们会持续观察是否有该问题

_Originally posted by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/issues/3404#issuecomment-1864173031_
            
当在一个浏览器 打开同一个ip地址的两个不同1panel时  没有登录的那一个面板就会无限返回登录页。清除缓存后可以登录成功，之前登录的会强制退出 在登录会出现无限返回登录页

**Comment by zhengkunwang223** [comment]
好的 我们验证一下 
话说为什么要在一个 IP 地址打开搞两个 1Panel 
是内网穿透吗？

**Comment by likesrt** [comment]
> 好的 我们验证一下 话说为什么要在一个 IP 地址打开搞两个 1Panel 是内网穿透吗？

比如我买了两台 nat 云服务器 刚好是同一个ip 就会出现这种情况。正巧我今天搞了两个nat服务器才发现
